data.sql

```diff
@@ -54,4 +54,85 @@ UPDATE animals SET owner_id = CASE
+((select id from vets where name like 'Stephanie Mendez'),
+(select id from species where name = 'Digimon')),
+((select id from vets where name like 'Stephanie Mendez'),
+(select id from species where name = 'Pokemon')),
+((select id from vets where name like 'Jack Harkness'),
+(select id from species where name = 'Digimon'));
+
+--- visits data
+INSERT INTO visits(animal_id, vet_id, date_of_visit) VALUES
+  ((select id from animals where name like 'Agumon'),
+    (select id from vets where name like 'William Tatcher'),
+    '2020-05-24'),
+  ((select id from animals where name like 'Agumon'),
+    (select id from vets where name like 'Stephanie Mendez'),
+    '2020-07-22'),
+  ((select id from animals where name like 'Gabumon'),
+    (select id from vets where name like 'Jack Harkness'),
+    '2021-02-02'),
+  ((select id from animals where name like 'Pikachu'),
+    (select id from vets where name like 'Maisy  Smith'),
+    '2020-01-05'),
+  ((select id from animals where name like 'Pikachu'),
+    (select id from vets where name like 'Maisy Smith'),
+    '2020-03-08'),
+  ((select id from animals where name like 'Pikachu'),
+    (select id from vets where name like 'Maisy Smith'),
+    '2020-05-14'),
+  ((select id from animals where name like 'Davimon'),
+    (select id from vets where name like 'Stephanie Mendez'),
+    '2021-05-04'),
+  ((select id from animals where name like 'Chamandar'),
+    (select id from vets where name like 'Jack Harkness'),
+    '2021-02-24'),
+  ((select id from animals where name like 'Plantmon'),
+    (select id from vets where name like 'Maisy Smith'),
+    '2019-12-21'),
+  ((select id from animals where name like 'Plantmon'),
+    (select id from vets where name like 'William Tatcher'),
+    '2020-08-10'),
+  ((select id from animals where name like 'Plantmon'),
+    (select id from vets where name like 'Maisy Smith'),
+    '2021-04-07'),
+  ((select id from animals where name like 'Squirtle'),
+    (select id from vets where name like 'Stephanie Mendez'),
+    '2019-09-29'),
+  ((select id from animals where name like 'Angemon'),
+    (select id from vets where name like 'Jack Harkness'),
+    '2020-10-03'),
+  ((select id from animals where name like 'Angemon'),
+    (select id from vets where name like 'Jack Harkness'),
+    '2020-11-04'),
+  ((select id from animals where name like 'Boarmon'),
+    (select id from vets where name like 'Maisy Smith'),
+    '2019-01-24'),
+  ((select id from animals where name like 'Boarmon'),
+    (select id from vets where name like 'Maisy Smith'),
+    '2019-05-15'),
+  ((select id from animals where name like 'Boarmon'),
+    (select id from vets where name like 'Maisy Smith'),
+    '2020-02-27'),
+  ((select id from animals where name like 'Boarmon'),
+    (select id from vets where name like 'Maisy Smith'),
+    '2020-08-03'),
+  ((select id from animals where name like 'Boarmon'),
+    (select id from vets where name like 'Stephanie Mendez'),
+    '2020-05-24'),
```

Comment:
- [ ] according to the provided data this should be Blossom instead of Boarmon

Reply:
Made the required adjustment 👍